### PR TITLE
Implement 'Quem Somos' page

### LINKED ISSUE #41: FE - Add Quem somos page
- Check list of names
- Add page

### PR DESCRIPTION
**Pull request by nancruz**
In this commit we implement the "Quem Somos" page.

image

Closes #41

### CODE REVIEW

File: app/javascript/components/about-us/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import JoinUs from "./JoinUs";
+import AboutUsFooter from "./AboutUsFooter";
+import rowsPeople from "../../../assets/images/rows-people.svg";
+
+export default function AboutUs() {
+    return (
+        <Layout>
+            <LayoutHeader />
+            <Layout.Content className="about-us-section">
+                <img src={rowsPeople} />
```
Comment (inesrosete) [inline app/javascript/components/about-us/index.js]: This image is not centered, check this image

image

**Review by inesrosete — COMMENTED** [review]
Do not forget to align scss files with #112.

File: app/javascript/components/about-us/index.js (hunk 1 above)
Comment (nancruz) [reply app/javascript/components/about-us/index.js]: You are right. I've tested it on the browser but forgot to add it to the css file.

Thanks

**Comment by rafaelcpalmeida** [comment]
It appears that you're missing some configs on your git. Is this intentional?

EDIT: you don't have the name neither the email configured for your user.

**Review by inesrosete — APPROVED** [review]
LGTM

**Pull request merged**